src/test/java/org/folio/rest/support/matchers/DomainEventAssertions.java

```diff
@@ -24,19 +24,17 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertEquals;
 
+import io.vertx.core.MultiMap;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
-import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
-
 import org.folio.services.kafka.KafkaMessage;
 
-import io.vertx.core.MultiMap;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
-
 public final class DomainEventAssertions {
+  private static final String NULL_INSTANCE_ID = "00000000-0000-0000-0000-000000000000";
```

Comment:
This is also defined in `CommonDomainEventPublisher`. Consider to define it just once.

Reply:
Done